src/nsm.h

```diff
@@ -57,10 +82,14 @@
 /*                                                           */
 /*         nsm_set_open_callback( nsm, cb_nsm_open, 0 );     */
 /*         nsm_set_save_callback( nsm, cb_nsm_save, 0 );     */
+/*         nsm_set_show_callback( nsm, cb_nsm_show, 0 );     */
+/*         nsm_set_hide_callback( nsm, cb_nsm_hide, 0 );     */
 /*                                                           */
 /*         if ( 0 == nsm_init( nsm, nsm_url ) )              */
 /*         {                                                 */
 /*             nsm_send_announce( nsm, "FOO", "", argv[0] ); */
+/*             nsm_check_wait(nsm, 200);                     */
```

Comment:
why is this added here?

Reply:
This allow the client to wait for the informations needed from the manager, eg, client_id.

Reply:
then please add a comment regarding that.
depending on how an application is made, this is quite optional.
(carla does not need this, for example)

also, this could take more than 200ms, so relying on it is problematic.

Reply:
I've increase the timeout to 100s, add a exit handler and a disclaimer to make clear that this part isn't needed most times.